packages/file_selector/file_selector/pubspec.yaml

```diff
@@ -14,10 +14,13 @@ flutter:
     platforms:
       web:
         default_package: file_selector_web
+      windows:
+        default_package: file_selector_windows
```

Comment:
Weirdest platform support of all Flutter plugins. I love it!

Thanks for bringing this over, congrats on the merge! 👏

Reply:
If only we'd started with the Windows implementation, we could have slowly added them in reverse alphabetical order.